cmp/filter/FilterChooserFieldSpec.js

```diff
@@ -4,157 +4,97 @@
  *
  * Copyright © 2021 Extremely Heavy Industries Inc.
  */
-import {HoistBase} from '@xh/hoist/core';
-import {FieldFilter, FieldType, genDisplayName, parseFieldValue} from '@xh/hoist/data';
+import {BaseFieldSpec} from '@xh/hoist/data/filter/BaseFieldSpec';
+import {FieldType, parseFieldValue} from '@xh/hoist/data';
 import {fmtDate} from '@xh/hoist/format';
 import {LocalDate} from '@xh/hoist/utils/datetime';
 import {stripTags, throwIf} from '@xh/hoist/utils/js';
-import {isFunction, isNil} from 'lodash';
+import {isFunction, isString, isNil} from 'lodash';
 
 /**
- * Defines field-level filtering options and provides metadata for presenting these options in
- * UI affordances, including data values available for suggestion if applicable and configured.
+ * In addition to the BaseFieldSpec, includes configuration for data values available for suggestion.
  *
  * Apps should NOT instantiate this class directly. Instead {@see FilterChooserModel.fieldSpecs}
  * for the relevant config to set these options.
  */
-export class FilterChooserFieldSpec extends HoistBase {
+export class FilterChooserFieldSpec extends BaseFieldSpec {
 
-    /** @member {string} */
-    field;
-
-    /** @member {string} */
-    displayName;
-
-    /** @member {string[]} */
-    ops;
+    /** @member {?Array} - data values available for suggestion. */
+    values;
 
-    /** @member {(boolean|FilterOptionSuggestValuesCb)} */
+    /** @member {(boolean|SuggestValuesCb)} */
     suggestValues;
 
     /** @member {boolean} */
     forceSelection;
 
-    /** @member {?Array} - data values available for suggestion. */
-    values;
-
-    /** @member {FilterOptionValueRendererCb} */
+    /** @member {FieldSpecValueRendererCb} */
     valueRenderer;
 
-    /** @member {FilterOptionValueParserCb} */
+    /** @member {FieldSpecValueParserCb} */
```

Comment:
These  jsdoc names are  wrong, no?  I think `FilterChooserValueParserCb` and FilterChooserRendererCb